tools/generate_leaderboard_plot.py

```diff
@@ -34,14 +34,17 @@ def main():
         "dino_vitb16_kaiko": "kaiko.ai - DINO ViT-B16 | TCGA",
         "dino_vitb8_kaiko": "kaiko.ai - DINO ViT-B8 | TCGA",
         "dino_vitl14_kaiko": "kaiko.ai - DINOv2 ViT-L14 | TCGA",
+        "bioptimus_h_optimus_0": "Bioptimus - H-optimus-0 | ViT-G14",
+        "prov_gigapath": "Prov-GigaPath | DINOv2 ViT-G14",
+        "histai_hibou_l": "hist.ai - hibou-L | DINOv2 ViT-B14",
```

Comment:
for the other FMs we introduced the syntax "<model> | <training data>". Lets keep this pattern for the three additional FMs?

Reply:
done